staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1100,7 +1100,8 @@ func (r Result) Into(obj runtime.Object) error {
 		return fmt.Errorf("serializer for %s doesn't exist", r.contentType)
 	}
 	if len(r.body) == 0 {
-		return fmt.Errorf("0-length response")
```

Comment:
There's no err or we'd have exited on line 1097

Reply:
fixed